datalad/metadata/search.py

```diff
@@ -25,6 +25,8 @@
 import sys
 from time import time
 
+from pkg_resources import EntryPoint, iter_entry_points
```

Comment:
let's move these two imports to the point of use.  `search` itself defines the Search interface so we better avoid heavy imports within this module signature

Reply:
OK